raffle.sh

```diff
@@ -42,7 +42,7 @@ draw growth/participants 1000
 # an emoji smile).
 draw diamond-exhibition-reshuffling/bonus-draw/participants 0 | (awk '!seen[$0]++' || :) | head -n 39
 
-draw toobins/jul-28/particpants 1
+draw toobins/jul-28/participants 1
```

Comment:
fix typo for an older draw, so we dont copy/paste the same typo

Reply:
should be `partypants`